project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.10.11
```

Comment:
why the dial back?

Reply:
Sorry I'll revert
For some reason this doesn't work for me, but only in explore